apps/server/src/modules/deletion-console/deletion-console.module.ts

```diff
@@ -4,12 +4,13 @@ import { ConfigModule } from '@nestjs/config';
 import { ConsoleModule } from 'nestjs-console';
 import { ConsoleWriterModule } from '@infra/console';
 import { createConfigModuleOptions } from '@src/config';
-import { DeletionClient } from '../client';
-import { getDeletionClientConfig } from '../client/deletion-client.config';
-import { BatchDeletionService } from '../services';
-import { BatchDeletionUc, DeletionExecutionUc } from '../uc';
+import { DeletionClient } from './deletion-client';
+import { getDeletionClientConfig } from './deletion-client/deletion-client.config';
 import { DeletionQueueConsole } from './deletion-queue.console';
 import { DeletionExecutionConsole } from './deletion-execution.console';
+import { BatchDeletionService } from './services';
+import { BatchDeletionUc } from './uc';
+import { DeletionExecutionUc } from '../deletion/uc';
```

Comment:
```suggestion
import { DeletionExecutionUc } from '@modules/deletion';
```
!!! Circular Dependency